.sequelizerc

```diff
@@ -0,0 +1,10 @@
+require('@babel/register');
```

Comment:
@avpaul shouldn't this conform to ES6 standards?

Reply:
Thanks! consider it done